### DISCUSSION TITLE
Can rnaspades be used to treat metatranscriptomic data?

### DISCUSSION THREAD

**Discussion by Holly202109** [ablab/spades#799] [Q&A]
### Is your feature request related to a problem? Please describe.

HI, thanks for the great tool. I am interested in rnaspades, I would like to know whether rnaspades can be used to treat metatranscriptomic data. Thank you.

**Comment by asl** [comment]
@Holly202109 Please do not create issues for the generic questions as you were advised. Tagging @andrewprzh

**Comment by andrewprzh** [comment]
Hi!

We have tested rnaSPAdes on metatranscriptomic data, but not a lot. I'd say it gives better results that metaSPAdes or usual SPAdes, and it's definitely safe to run it. You may expect a bit of redundancy in the resulting contigs since rnaSPAdes tries to reconstruct alternative isoforms typical for eukaryotic genomes.

Best
Andrey

**Comment by Holly202109** [comment]
Thanks a lot, Andrey. I got it and will use it to assemble my metatranscriptomic reads.

Do you think I should remove the rRNA reads before the assembly step? or should include them?




Best regards

Liping




-----原始邮件-----
发件人:"Andrey Prjibelski" ***@***.***>
发送时间:2021-06-25 19:25:34 (星期五)
收件人: ablab/spades ***@***.***>
抄送: Holly202109 ***@***.***>, Mention ***@***.***>
主题: Re: [ablab/spades] Can rnaspades be used to treat metatranscriptomic data? (#799)






Hi!

We have tested rnaSPAdes on metatranscriptomic data, but not a lot. I'd say it gives better results that metaSPAdes or usual SPAdes, and it's definitely safe to run it. You may expect a bit of redundancy in the resulting contigs since rnaSPAdes tries to reconstruct alternative isoforms typical for eukaryotic genomes.

Best
Andrey

**Reply by andrewprzh** [reply]
You may try removing rRNA either from reads or from contigs, my guess is that the results will be quite similar. You may try both strategies if you have time :)
Unfortunately, I cannot say for sure which one is better.
